web/themes/custom/fmcg_material/sass/_form.scss

```diff
@@ -11,19 +11,26 @@ form {
     word-wrap: normal;
   }
 
+  .field--name-description-summary .filter-wrapper {
```

Comment:
Is this only desired for the description summary field? OR for all formatted text fields?

Reply:
As discussed, after not being able to find where this rule is in effect, I've deleted it.